feat: add beagle flutter documentation (#717)
ZupIT/beagle-docs: content/en/resources/customization/beagle-for-flutter/development.md

---
title: Development
weight: 5
description: >-

>   You will find here information of how the development of the Beagle Flutter is going.

Reviewer: ```suggestion
 In this section, you will find information on how Beagle Flutter's developement is going.
```

---

---

## Current development status
This is our current development status. Please help us get to a stable version!

To contribute to this project, please use [this repository](https://github.com/ZupIT/beagle).

### Core features

> Almost done. Renderization, context, custom components and custom actions all work! Only the layout engine which is based on flexbox is partially applied. The styling is not implemented either.

Reviewer: ```suggestion
The core features are almost done, see below what is already working:
- Renderization;
- Context;
- Custom components
- Custom actions. 
The layout engine is based on flexbox and it is partially applied. The styling is not implemented either.
```
